wikiweaver-web/index.html

```diff
@@ -45,7 +45,12 @@
             <div class="text">Start a race from</div>
             <input id="start-page-input" class="box text" placeholder="Gingerbread"></input>
             <div class="text">to</div>
-            <input id="goal-page-input" class="box text" placeholder="League of Legends"></input>
+            <div id="goal-page-container">
+                <button id="show-goal-text" class="button box" onclick="ToggleGoalSummary()">intro</button>
+                <input id="goal-page-input" class="box text" placeholder="League of Legends"></input>
```

Comment:
something happened with the width here
![image](https://github.com/user-attachments/assets/5ca59cd9-f315-415f-8250-525beb6a20b0)